code/modules/research/xenobiology/vatgrowing/microscope.dm

```diff
@@ -6,14 +6,21 @@
 	var/obj/item/petri_dish/current_dish
 
 /obj/structure/microscope/attacked_by(obj/item/I, mob/living/user)
-	if(!istype(I, /obj/item/petri_dish))
+	if(I.tool_behaviour == TOOL_WRENCH)
```

Comment:
put this in wrench_act

Reply:
Also you should probably use the default wrench helper here, since it's the same thing.